README.md

```diff
@@ -6,7 +6,7 @@ Brought to you by UMass Medical School, Weng Lab.
 
 ## Installation
 
-For npm use: `npm install bigwig-reader --save`
+For npm use: `npm install bigwig-reader`
 
 For yarn use: `yarn add bigwig-reader`
```

Comment:
genomic-reader here too

Reply:
@hpratt Done. Thanks for catching this. I fixed this.  

While I got your attention, when I do install genomic-reader with yarn or npm, I do not get the dist folder. Could it be something wrong with my local machine?

Reply:
hmm that was the exact issue I was trying to fix, are you forcing v1.4.7? with "genomic-reader": "1.4.7" in package.json it does have the dist folder on my machine

Reply:
@hpratt Thanks for confirming the dist issue. I was using v1.4.1. I will try try the new updates later to see if dist is automatically added. Thanks for fixing the issue.